modules/rubiconBidAdapter.js

```diff
@@ -525,7 +525,7 @@ export const spec = {
     }
 
     // Send multiformat data if requested
-    if (params.bidonmultiformat === true && deepAccess(bidRequest, 'mediaTypes') && Object.keys(bidRequest.mediaTypes).length > 1) {
+    if (params.bidOnMultiFormat === true && deepAccess(bidRequest, 'mediaTypes') && Object.keys(bidRequest.mediaTypes).length > 1) {
```

Comment:
Interesting the param below this line was already camel case, possible bug @robertrmartinez ?